serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/utils/ProtobufUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.amazonaws.services.schemaregistry.utils;
+
+import com.amazonaws.services.schemaregistry.utils.apicurio.FileDescriptorUtils;
+import com.google.protobuf.DescriptorProtos;
+
+public final class ProtobufUtils {
```

Comment:
Minor: Utils can attract multiple unrelated classes. We can call this `ProtobufSchemaParser`

Reply:
done